recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
 outputs:
```

Comment:
Do you by chance know why these are separated as such? It seems as though `redis` and `filelock` would work well together. However at a glance, it seems like the functionality is being separated into two different packages, which does not seem very nice for our users, as they would have to install 3 separate packages. Maybe not now, but in the future we could look at adding the optional dependencies into `run_constrained` or something like that.

Reply:
NB that this means that users already using `cachecontrol-with-redis` or `cachecontrol-with-filecache` wouldn't receive any updates to it. It would be something like an API break